webpack.config.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
         theme: "theme",
         admin: "admin",
         oembed: 'oembed',
+        henri: 'js/henri.js',
```

Comment:
I think you can safely put it into the parent folder so it has the same convention than other files

Reply:
:+1: